syntax = "proto3";

package myservice;

message MyRequest {
  string name = 1;
}

message MyResponse {
  string greeting = 1;
}

service MyService {
  rpc SayHello (MyRequest) returns (MyResponse);
}
